dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/environmental/subcategories/SfdrEnvironmentalEnergyPerformance.kt

```diff
@@ -0,0 +1,52 @@
+package org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories
+
+import org.dataland.datalandbackend.model.DataPointWithUnit
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the subcategory "Energy performance" belonging to the category "Environmental" of the sfdr framework.
+ */
+data class SfdrEnvironmentalEnergyPerformance(
+    val renewableEnergyProduction: DataPointWithUnit<BigDecimal>? = null,
```

Comment:
Same thing here as for `SfdrEnvironmentalEmissions.kt`:  We could make the devtools know "DataPointWithUnit" and then the autogenerated models would already in clude it.

Reply:
Now autogenerated

Reply:
👍